dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -262,6 +262,17 @@ public void Dispose()
         GC.SuppressFinalize(this);
     }
 
+    /// <summary>
+    /// Returns the name and description of the function.
+    /// </summary>
+    /// <returns></returns>
+    public override string ToString()
+    {
+        string s = this.IsSemantic ? "Semantic" : "Native";
+
+        return $"{this.Name} \t {s} \t- {this.Description}";
```

Comment:
That's a strange mix of tabs and spaces.  Also, you might want to specify from which skill the function is.